db/migrate/20250519155554_create_new_api_tokens.rb

```diff
@@ -0,0 +1,15 @@
+class CreateNewAPITokens < ActiveRecord::Migration[8.0]
+  def change
+    create_table :new_api_tokens do |t|
+      t.references :tokenable, polymorphic: true, index: true
+
+      t.string :hashed_token, null: false
+      t.string :description
```

Comment:
Could `:description` be made mandatory?  Would make like a bit easier when looking at data perhaps if we by default include the `LeadProvider.name` when generating for example?  Is it overkill - what do you think?

Reply:
Yeah agree!